components/helm-broker/internal/broker/deprovision.go

```diff
@@ -148,6 +154,7 @@ func (svc *deprovisionService) do(ctx context.Context, iID internal.InstanceID,
 	}
 
 	if err := svc.operationUpdater.UpdateStateDesc(iID, opID, opState, &opDesc); err != nil {
-		// TODO: create event from broker and log as we are not able to propagate failure to service catalog
+		svc.log.Errorf("Cannot update state for instance [%s]: [%v]\n", iID, err)
```

Comment:
what is a purpose of `\n`? AFAIK In logs you do not need that

Reply:
removed, I copied it from the application broker I think